hack/pre-release.sh

```diff
@@ -63,11 +64,13 @@ update_WMCO_version() {
   echo "Updating the skipRange version in $cluster_service_version_yaml to $updated_version"
   # Example: matches olm.skipRange: '>=8.0.0 *<9.0.0*'
   sed -i "s/<$version/<$updated_version/g" $cluster_service_version_yaml
-
-  echo "Updating previous skipRange version in $cluster_service_version_yaml to $version"
-  # Example: matches olm.skipRange: '*>=8.0.0* <9.0.0'
-  sed -i "s/>=$last_version/>=$version/g" $cluster_service_version_yaml
- 
+  
+  if [[ $xyz_stream == "x" ]]; then 
+    echo "Updating previous skipRange version in $cluster_service_version_yaml to $version"
+    # Example: matches olm.skipRange: '*>=8.0.0* <9.0.0'
```

Comment:
Please call out what you are replacing in addition. You might want to do this in other places too in a separate PR.